dex/testing/dcr/harness.sh

```diff
@@ -89,19 +89,53 @@ chmod +x "${NODES_ROOT}/harness-ctl/mine-beta"
 # Reorg script
 cat > "${NODES_ROOT}/harness-ctl/reorg" <<EOF
 #!/bin/sh
+REORG_NODE="alpha"
+VALID_NODE="beta"
+REORG_DEPTH=1
+
+if [ "\$1" = "beta" ]; then
+  REORG_NODE="beta"
+  VALID_NODE="alpha"
+  REORG_DEPTH=3
+fi
```

Comment:
I've had a need to cause reorg on either nodes while testing the FindRedemption feature on simnet, decided to add it here for convenience. I ultimately plan to use this feature in the simnet trading tests.